docusearch/templates/docusearch/detail.html

```diff
@@ -1,14 +1,17 @@
 {% extends "base.html" %}
 
-
 {% block body %}
 
 <div class="container">
+    <section class="breadcrumbs">
+        <a href="{{url('home')}}">Home &gt;</a>
+        <a href="{{url('doc_search')}}?q={{request.GET.q}}">Search &gt;</a>
```

Comment:
Should we add the query to the link text when possible? Currently, the user might not know when clicking on `search` will return them to their original search or a blank search.